cpg-analysis/src/main/kotlin/de/fraunhofer/aisec/cpg/analysis/fsm/DFAOrderEvaluator.kt

```diff
@@ -300,12 +302,28 @@ open class DFAOrderEvaluator(
         seenStates: MutableSet<String>
     ): List<Node> {
         val outNodes = mutableListOf<Node>()
-        outNodes += node.nextEOG
+        // outNodes += node.nextEOG
+        outNodes +=
+            PropertyEdge.unwrap(
+                node.nextEOGEdges.filter { e ->
+                    e.getProperty(Properties.UNREACHABLE) == null ||
```

Comment:
Should we somehow make this configurable or do we *always* want to filter unreachable code?